client/app/telecom/telephony/service/assist/orders/telecom-telephony-service-assist-orders.html

```diff
@@ -16,18 +16,16 @@ <h1 data-translate="telephony_line_assist_orders_title"></h1>
 
         <div class="telecom-section-content">
             <div class="table-responsive-vertical">
-                <table class="table table-pretty table-hover table-striped no-space">
+                <table class="table table-pretty table-hover table-striped m-0">
```

Comment:
```html
<table class="table"
       data-ng-class="{
           'table-hover table-striped': OrdersCtrl.orders.raw > 1
       }">
```

Reply:
They are 12 tables in the same condition, we should fix those at the same time in another pull request.
P.S. I would prefer a solution using strictly CSS for those corner cases...